source/variables/typography.css

```diff
@@ -0,0 +1,13 @@
+:root {
+	/*The base font-size for the root html element.  The value represents its pixel equivalent, e.g. 16 = 16px.*/
+    --base--font-size: 16;
+
+    /*The percentage of the font-size to be used on the root html element.*/
+    --base--font-percentage: calc(calc(var(--base--font-size) / 16) / 100%);
+    
+    /*The base line height for the root html element.*/
+    --base--line-height: 1.5;
+
+    /*The base font-family for the root html element.*/
+    --base--font-family: sans-serif;
+}
```

Comment:
There is a different semantic meaning for a single `-` verses a double `--`. 

`--` symbolizes a variation, where `-` symbolizes a sub variable. These should all use a single `-`, where something like z-index is more appropriate to use double `--`

Then if i have to create a variation, it's clear what's happening:

```
--base-font-size: 16;
--base-font-size--modal: 24
```